library/cartridge_get_shortened_fact_sets.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+
+from ansible.module_utils.helpers import Helpers as helpers
+
+argument_spec = {
+    'hostvars': {'required': True, 'type': 'dict'},
+}
+
+sets_list = {
+    'validate_config': [
+        'all_rw',
+        'cartridge_app_config',
+        'cartridge_app_name',
+        'cartridge_auth',
+        'cartridge_bootstrap_vshard',
+        'cartridge_cluster_cookie',
+        'cartridge_conf_dir',
+        'cartridge_configure_systemd_unit_files',
+        'cartridge_configure_tmpfiles',
+        'cartridge_custom_steps',
+        'cartridge_custom_steps_dir',
+        'cartridge_data_dir',
+        'cartridge_defaults',
+        'cartridge_enable_tarantool_repo',
+        'cartridge_failover',
+        'cartridge_failover_params',
+        'cartridge_install_dir',
+        'cartridge_install_tarantool_for_tgz',
+        'cartridge_instances_dir',
+        'cartridge_keep_num_latest_dists',
+        'cartridge_memtx_dir_parent',
+        'cartridge_multiversion',
+        'cartridge_package_path',
+        'cartridge_remove_temporary_files',
+        'cartridge_run_dir',
+        'cartridge_scenario',
+        'cartridge_systemd_dir',
+        'cartridge_tmpfiles_dir',
+        'cartridge_vinyl_dir_parent',
+        'cartridge_wait_buckets_discovery',
+        'cartridge_wal_dir_parent',
+        'config',
+        'edit_topology_timeout',
+        'expelled',
+        'failover_priority',
+        'instance_discover_buckets_timeout',
+        'instance_start_timeout',
+        'replicaset_alias',
+        'restarted',
+        'roles',
+        'stateboard',
+        'vshard_group',
+        'weight',
+        'zone',
```

Comment:
Seems that if somebody add a new variable and forget to add it there, it wouldn't be validated.

Reply:
Good comment, I missed some of the variables. Added a test for this.